.github/dependabot.yml

```
@@ -13,7 +13,7 @@ updates:
     rebase-strategy: "disabled"
     ignore:
       - dependency-name: "@types/node"
-        versions: ["17", "18"]
+        versions: [">=20"]
```

Should this be `<=20` since won't this ignore any 20.x versions?

---

Ah, yeah, that should be:
```suggestion
        versions: [">=21"]
```
to ignore 21+. Fat fingered that version 🤦